### ISSUE TITLE
lampe uses wrong color escape codes for white and ltwhite, (was: "solarized ..")

### ISSUE THREAD

**Issue by bserem** [poinck/lampe#39]
http://ethanschoonover.com/solarized is a very friendly to the eyes color set for terminals, editors etc.

Currently, lampe mixes some of its text (mainly the press h for help text) with the default solarized background.
Given that solarized is over-popular, and its colors are "set in stone" and can't change, I suppose the fix should come from lampe.

**Comment by poinck** [comment]
Hey, thx for this report. Can you please attached a screenshot with the color-scheme-problem? I have no working gnome-terminal to test "solarized", I suppose this is not the same color-scheme, anyway.

**Comment by bserem** [comment]
![lampe](https://cloud.githubusercontent.com/assets/520237/21289089/63b4b6ba-c49c-11e6-918e-a313a15e9359.jpg)
vs
![lampe_good](https://cloud.githubusercontent.com/assets/520237/21289098/8de8db78-c49c-11e6-86c9-3c73b36a2a70.jpg)

You can slightly see the name of the lamps next to their numbers.
Help text is not visible.

Btw you do not need Gnome. I use XFCE.
The solarized color pallete is compatible with many terminals and editors.
Regardless of this issue you might like it yourself, give it a try!

**Comment by poinck** [comment]
Maybe I can change the escape-sequences for colors that work for light and dark backgrounds. The solution could be that users with light background will need to set an option in "~/.lamperc". I will consider both solutions and let you know.

But, I will not try "solorized", it has not enough contrast; I have my own .Xresources for urxvt set to my needs.

**Comment by poinck** [comment]
You can now try a fix:

```.sh
git fetch
git checkout issue39-color_escapes
git pull
./lampe
```

**Comment by bserem** [comment]
Solarized Dark looks really good.
![screenshot_2016-12-18_18-32-37](https://cloud.githubusercontent.com/assets/520237/21294861/614bc774-c550-11e6-8d49-d36da2d0d1bc.png)

Solarized Light not so much:
![screenshot_2016-12-18_18-33-39](https://cloud.githubusercontent.com/assets/520237/21294867/8254edb0-c550-11e6-9057-bd99cba3bf4e.png)

I'll try to see if I can improve things, following your commit.

Thanks

**Comment by poinck** [comment]
Interesting, "solarized light" treats ltwhite as background color. My personal opinion is, that this is wrong; it shouldn't do that. 

Anyway, I have decided to have it not look ugly with "solarized light" (: I will commit the change shortly.

![2016-12-20-192753_722x434_scrot](https://cloud.githubusercontent.com/assets/1297960/21362915/9600b880-c6ea-11e6-902b-dcd445ec0a54.png)

**Comment by poinck** [comment]
You can pull again and test. I just forgot to add "#39" to the commit-message.

**Comment by bserem** [comment]
![lampe_solarized_light](https://cloud.githubusercontent.com/assets/520237/21364417/77d87708-c6f9-11e6-8fc2-c888f397ad55.png)
I like how you approach other peoples bugs :+1: 

It looks really good now! I suppose we can close this issue.
Thanks

**Comment by poinck** [comment]
Yes, thanks for testing. (:

This will be merged to master soon and released as version 1.1.8 together with fix for issue #40 .
